dex/src/main/scala/com/wavesplatform/dex/cli/WavesDexCli.scala

```diff
@@ -226,15 +224,15 @@ object WavesDexCli extends ScoptImplicits {
                   _ <- cli.log(
                     s"""
                       |Passed arguments:
+                      |  DEX REST API          : ${args.dexRestApi}
```

Comment:
If it is empty, we doesn't know, where we are connecting to. Consider to determine the final `dexRestApi` here.

Reply:
It would be printed below after SuperConnector creation